cli/src/config.rs

```diff
@@ -158,6 +164,8 @@ impl Default for Settings {
             output_format: OutputFormat::Table,
             show_progress: false,
             max_display_rows: 40,
+            max_col_width: 0,
+            max_width: 0,
```

Comment:
can we set it to be terminal width ?

Reply:
Duckdb is this: if max width is 0, will use terminal width. But I don't know how to get the terminal width in rust.

Reply:
Done